src/server/api/schema.js

```diff
@@ -1080,7 +1080,7 @@ const rootMutations = {
           // run interaction step handler
           try {
             const handler = require(`../../integrations/action-handlers/${interactionStepAction}.js`);
-            handler.processAction(
+            await handler.processAction(
```

Comment:
I'm sorry I missed the `await` from my previous review -- but why do we want to `await` here?  This is blocking the response to the texter....
If this is just for easier testing, then please use the pattern:
```
const handlerResult = handler.processAction(...)
if (global.TEST_ENVIRONMENT) {
   await handlerResult;
}
```